oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticCustomAnalyzer.java

```diff
@@ -257,14 +257,17 @@ private static <FD> LinkedHashMap<String, FD> loadFilters(NodeState state,
 
             Map<String, Object> args = convertNodeState(child, transformers, content);
 
-            if (name.equals("word_delimiter")) {
+            if (name.equals("standard")) {
+                // OAK-11638 ignore standard token filter
```

Comment:
good catch! I would just add a comment to the ES 7 release note where this was removed
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/breaking-changes-7.0.html#standard-filter-removed